_data/beyond-sidebar.yml

```diff
@@ -559,9 +576,12 @@
             - title: Capture payment
               link: https://api-docs.beyondshop.cloud/ng-order-order-public-api.html#resources-payment-process-capture
               id: capture_payment
-            - title: Create pickup processes
+            - title: Create pickup process
               link: https://api-docs.beyondshop.cloud/ng-order-order-public-api.html#resources-pickup-processes-create
               id: create_pickup_processes
```

Comment:
Is it ok to keep the id unchanged when the title changes?

Reply:
Technically, yes.

Reply:
I adapted the ID as well.